wpengine/faustjs
# Merl 1306 multiple queries

## Tasks


## Description

This PR adds a new possible field on a Faust WordPress Template called `queries`. This property allows you to supply multiple queries and variables to one Faust Template.



## Related Issue(s):



## Testing

1. Checkout this branch
2. Build the packages `npm run build`
3. Start the example project `npm run dev -w @faustwp/getting-started-example`
4. Visit a `Single` post template and notice that the multiple queries are being resolved as expected.
5. Visit a `Page` template and notice that the singe query works as intended.
6. Visit a mix of both single posts and pages; trying previews for each, CSR, and regular SSG to look for any irregularities.
7. Try to break it. There are a number of changes in core Faust features here so we want to make sure there is no regression.

New unit tests for WordPressTemplate have been added.



## Screenshots



## Documentation Changes



## Dependant PRs
